docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
Yeah. We've bikeshed a ton (mostly in person) and we just don't have a catchy name. Process manager was the most apt definition but is a horrible name. i really wish we could come up with a clever name that has no real meaning in tech previously, like "saga" was (before someone coined it ". I DUNNO........ 😖